css-three-column-layout/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Three Column Layout</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <section class="container">
+      <h2>Designed for life science. Optimized for SaMD and Digital Health.</h2>
+      <div class="row">
+        <div class="column-one-third">
+          <div class="row">
+            <div class="white-circle">
+              <img
+                src="./images/merge.png"
+                alt="A flowchart-esque group of three combined into one moving forward with a directional arrow" />
+            </div>
+          </div>
```

Comment:
This inner `row` is a little excessive. All you are using this for is flexing your `div` that holds the `img` and centering it. Instead, why not make your `column-one-third` _also_ a flex container? It would save on one `div`, you would just need to apply similar classes to your `column-one-third`

You could also center your image but just using margin too and not make your `column-one-third` another flex container. Couple of options for you here but I would remove the excess `div`.

Reply:
Thanks! I was thinking uni-directionally! I took your advice and made the columns flex containers and used align-items as you no doubt thought I should.